src/app/routes/HomePage.vue

```diff
@@ -103,23 +66,17 @@ export default {
 
   watch: {
     searchResult(value) {
-      if (value.items.length) {
-        if (!this.isLazyEventListenerActive) {
-          this.addLazyEventListener();
-        }
-
-        setTimeout(() => {
-          this.lazyLoad();
-        }, 0);
-      }
+      value.items.length && setTimeout(this.lazy.update);
```

Comment:
When you are using a lot setTimeout it usually means that's something wrong )

Reply:
> When you are using a lot setTimeout it usually means that's something wrong )

Function could be called before items render, I know something about event loop and it helped

Reply:
Yeah, it helped ) But it's kind of hack )
Maybe you can find a way to be sure that it will run after render
Why? - setTimeout will not guarantee you that it will run after render
That can strike back on tests or on ssr, you will never know )